src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
I don't know what we should choose, but I find that aligning with the top token often breaks over time and prefer to just align it one block more than the wrapping brackets (could be done with `AlignAfterOpenBracket: DontAlign`). What do you think?